content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -103,13 +103,20 @@ Setting the ingress IP depends on the cluster provider:
     $ export INGRESS_HOST=$(minikube ip)
     {{< /text >}}
 
+1.  _IBM Cloud Kubernetes Service:_
+
+    {{< text bash >}}
+    $ ibmcloud ks workers --cluster <cluster-name or id>
+    $ export INGRESS_HOST=<public IP of one of the worker nodes>
+    {{< /text >}}
```

Comment:
```suggestion
    {{< text bash >}}
    $ ibmcloud ks workers --cluster cluster-name-or-id
    $ export INGRESS_HOST=public-IP-of-one-of-the-worker-nodes
    {{< /text >}}
```

Reply:
Actually you might want to move this entry up one more (ahead of minikube), so GKE and IBM Cloud are the first 2.